dlt/destinations/impl/filesystem/filesystem.py

```diff
@@ -872,7 +872,7 @@ def load_open_table(self, table_format: TTableFormat, table_name: str, **kwargs:
             return DeltaTable(table_location, storage_options=storage_options)
         else:
             raise NotImplementedError(
-                f"Cannot load tables in {table_format} format in filesystem destination."
+                f"Can't load tables using {table_format=:} using `filesystem` destination."
```

Comment:
```suggestion
                f"Can't load tables using `{table_format=:}` with `filesystem` destination."
```
or other way. but 2x using sounds odd. (also backticks)